python/ray/scripts/scripts.py

```diff
@@ -703,38 +713,49 @@ def up(cluster_config_file, min_workers, max_workers, no_restart, restart_only,
             logger.info("Error downloading file: ", e)
     create_or_update_cluster(cluster_config_file, min_workers, max_workers,
                              no_restart, restart_only, yes, cluster_name,
-                             no_config_cache)
+                             no_config_cache, log_old_style, log_color,
+                             verbose)
 
 
 @cli.command()
 @click.argument("cluster_config_file", required=True, type=str)
+@click.option(
+    "--yes",
+    "-y",
+    is_flag=True,
+    default=False,
+    help="Don't ask for confirmation.")
 @click.option(
     "--workers-only",
     is_flag=True,
     default=False,
     help="Only destroy the workers.")
+@click.option(
+    "--cluster-name",
+    "-n",
+    required=False,
+    type=str,
+    help="Override the configured cluster name.")
 @click.option(
     "--keep-min-workers",
     is_flag=True,
     default=False,
     help="Retain the minimal amount of workers specified in the config.")
 @click.option(
-    "--yes",
-    "-y",
-    is_flag=True,
-    default=False,
-    help="Don't ask for confirmation.")
+    "--log-old-style", is_flag=True, default=False, help=("Use old logging."))
```

Comment:
let's default this to True and feature flag it for 1 release

Reply:
done!